drone/client.go

```diff
@@ -218,9 +230,9 @@ func (c *client) BuildLast(owner, name, branch string) (*Build, error) {
 
 // BuildList returns a list of recent builds for the
 // the specified repository.
-func (c *client) BuildList(owner, name string) ([]*Build, error) {
+func (c *client) BuildList(owner, name string, opts *ListOptions) ([]*Build, error) {
```

Comment:
the only suggestion I have is to remove pointer to reduce possibility of a nil pointer when accessing `opts.Page`

```
-opts *ListOptions
+opts ListOptions
```

Reply:
Thanks for your review, I addressed the comment.